src/MoBi.CLI.Core/ApplicationStartup.cs

```diff
@@ -64,17 +68,28 @@ public static void Start()
          container.Register<IFileLocker, FileLocker>(LifeStyle.Singleton);
          container.Register<IMoBiConfiguration, IApplicationConfiguration, MoBiConfiguration>(LifeStyle.Singleton);
          container.Register<IMoBiXmlSerializerRepository, MoBiXmlSerializerRepository>(LifeStyle.Singleton);
-         container.Register<ICoreUserSettings, UserSettings>();
+         container.Register<IUserSettings, IPresentationUserSettings, OSPSuite.Core.ICoreUserSettings, MoBi.Core.ICoreUserSettings, UserSettings>(LifeStyle.Singleton);
+         container.Register<IApplicationSettings, OSPSuite.Core.IApplicationSettings, ApplicationSettings>(LifeStyle.Singleton);
          container.Register<ISpatialStructureDiagramManager, SpatialStructureDiagramManager>();
          container.Register<IMoBiReactionDiagramManager, MoBiReactionDiagramManager>();
          container.Register<ISimulationDiagramManager, SimulationDiagramManager>();
+         container.Register<ISettingsPersistor<IUserSettings>, UserSettingsPersistor>();
+         container.Register<ISettingsPersistor<IApplicationSettings>, ApplicationSettingsPersistor>();
+
+         
 
          register.PerformMappingForSerializerIn(container);
          initializeDimensions(container);
-         InitCalculationMethodRepository(container);
+         initCalculationMethodRepository(container);
+
+         var userSettingsPersistor = container.Resolve<ISettingsPersistor<IUserSettings>>();
+         userSettingsPersistor.Load();
+
+         var applicationSettingsPersistor = container.Resolve<ISettingsPersistor<IApplicationSettings>>();
+         applicationSettingsPersistor.Load();
```

Comment:
The `IUserSettings` and `IApplicationSettings` were registered but never loaded. So I needed to register the persistors and then make sure they are loaded. That means you set your PK-Sim path in the MoBi GUI. We could include a command line parameter too for CLI.

I'll make an issue for it.